### PR TITLE
docs: added a guidelines section

### PR DESCRIPTION
**Pull request by pim-simons**
Added a guidelines section to include best practices, for now this only includes installation.

Closes https://github.com/arcus-azure/arcus.scripting/issues/284

### CODE REVIEW

File: docs/preview/02-Guidelines/Installation.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+To have access to the Arcus Scripting features, you have to import the modules.
+The best practice for usage in your build and release pipelines is to use the following commands:
+
+``` powershell
+PS> Set-PSRepository -Name PSGallery -InstallationPolicy Trusted
+PS> Install-Module -Name Arcus.Scripting.{Module} -AllowClobber
+```
+
+This drastically improves performance over using the `-Force` parameter and as such, usage of the `-Force` parameter is not recommended.
```
Comment (stijnmoreels) [inline docs/preview/02-Guidelines/Installation.md:16]: Great to have this! 👍

**Review by stijnmoreels — APPROVED** [review]
💃

**Comment by stijnmoreels** [comment]
Should we then remove this from the DevOps page? Or do you think that since it's mostly in DevOps scenario's that this problem occur, that we would leave it there?

**Comment by pim-simons** [comment]
> Should we then remove this from the DevOps page? Or do you think that since it's mostly in DevOps scenario's that this problem occur, that we would leave it there?

I would keep it in, people can overlook the guidelines page and by having it in the DevOps page we make sure the correct way is used.

**Pull request merged**